src/sentry/utils/suspect_resolutions_releases/get_suspect_resolutions_releases.py

```diff
@@ -0,0 +1,51 @@
+from typing import Sequence
+
+from sentry import features
+from sentry.models import Group, GroupRelease, GroupStatus, Project, Release, ReleaseProject
+from sentry.signals import release_created
+from sentry.tasks.base import instrumented_task
+from sentry.utils.suspect_resolutions_releases import ALGO_VERSION, analytics
+
+
+@release_created.connect(weak=False)
+def record_suspect_resolutions_releases(release, project, **kwargs) -> None:
```

Comment:
Looks like the `release_created` signal doesn't provide `project` here:
https://github.com/getsentry/sentry/blob/c5bcf354ae7646889c8e8dea969605b3d4560acc/src/sentry/signals.py#L103

Might make sense to remove `project` here and handle the querying in `get_suspect_resolutions_releases`

Reply:
Is there a way to query for a specific `project` given a `release`? I don't think querying `ReleaseProject` works because there could be multiple projects associated with a release

Reply:
https://github.com/getsentry/sentry/blob/7c5f68ef587b9231bfcdf7b6adaca2cf50c2766e/src/sentry/models/release.py#L430-L432

This attr includes all the projects associated for that specific release. I think its better to remove `project` from the method signature altogether and try to piece together a query in the `get_suspect_resolutions_releases` task creation step.